shell.nix

```diff
@@ -1,12 +1,27 @@
-{ pkgs ? import <nixpkgs> {} }:
-with pkgs;
+{ pkgs ? import <nixpkgs> {}}:
+let minikubePinned =
+let
+  version = "1.25.2";
+  src = pkgs.fetchFromGitHub {
+    owner = "kubernetes";
+    repo = "minikube";
+    rev = "v${version}";
+    sha256 = "sha256-WIk4ibq7jcqao0Qiz3mz9yfHdxTUlvtPuEh4gApSDBg=";
+  };
+  in (pkgs.minikube.override rec {
+    buildGoModule = args: pkgs.buildGoModule.override {} (args // {
+      inherit src version;
+      vendorSha256 = "sha256-8QqRIWry15/xwBxEOexMEq57ol8riy+kW8WrQqr53Q8=";
+    });
+  });
+in with pkgs;
```

Comment:
When this built locally it emitted the following:
```
go build  -tags "" -ldflags="-X k8s.io/minikube/pkg/version.version=v1.25.2 -X k8s.io/minikube/pkg/version.isoVersion=v1.25.2 -X k8s.io/minikube/pkg/version.gitCommitID=v1.27.0 -X k8s.io/minikube/pkg/version.storageProvisionerVersion=v5" -o out/minikube k8s.io/minikube/cmd/minikube
```
Which seems a little odd with `gitCommitID=v1.27.0`

Reply:
oh weird. I'll take a look.